src/extension.ts

```diff
@@ -143,7 +145,7 @@ export function activate(context: vscode.ExtensionContext): void {
 	context.subscriptions.push(
 		vscode.commands.registerCommand(
 			`${SETTINGS_SECTION_ID}.start.externalDebugPreview.atFile`,
-			(file?: vscode.Uri | string, options?: openFileOptions) => {
+			async (file?: vscode.Uri | string, options?: openFileOptions) => {
```

Comment:
is it ok to register an async function like this in registerCommand?

Reply:
Yes